entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -ex
+
+ 
+/root/game_backend/_build/prod/rel/game_backend/bin/game_backend start
```

Comment:
I think that since the application is still `:dark_worlds_server` the path should be:

```suggestion
/root/game_backend/_build/prod/rel/dark_worlds_server/bin/dark_worlds_server start
```